packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-configs.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { POO_STATUSES } from './constants/constants';
+import VerificationStatusActionButton from 'Components/verification-status-action-button';
+
+export const getPOOStatusMessages = (status: typeof POO_STATUSES[keyof typeof POO_STATUSES]) => {
+    const resubmitButton = (onClick?: () => void) => (
+        <VerificationStatusActionButton button_text={<Localize i18n_default_text='Try again' />} onClick={onClick} />
+    );
+
+    const titles: Record<typeof status, React.ReactElement> = {
+        none: <Localize i18n_default_text="Your proof of ownership isn't required." />,
+        pending: <Localize i18n_default_text='We’ve received your proof of ownership.' />,
+        rejected: <Localize i18n_default_text='Proof of ownership verification failed' />,
+        verified: <Localize i18n_default_text='Proof of ownership verification passed.' />,
+    };
+
+    const descriptions: Record<typeof status, React.ReactElement | null> = {
```

Comment:
For Localize to work properly, Shouldnt this be a function?

Reply:
It is a const defined inside the function. Will it cause an issue? @amina-deriv ?

Reply:
@amina-deriv , we grab these values with the function itself, it is just objects inside this function. is it ok?